Machine.h

```diff
@@ -45,6 +45,8 @@ typedef struct Machine_ {
    struct timeval realtime;   /* time of the current sample */
    uint64_t realtimeMs;       /* current time in milliseconds */
    uint64_t monotonicMs;      /* same, but from monotonic clock */
+   
+   int iterationsRemaining;
```

Comment:
I am not sure if this should be of type `int64_t` or if `int` is good enough.

Reply:
The bit-sized types are preferred, thus I'd suggest `int64_t`.